src/components/ProductFilters/ProductFilters.jsx

```diff
@@ -0,0 +1,313 @@
+                type="checkbox"
+                value="men & women"
+                checked={productState.category.includes("men & women")}
+                onChange={(e) =>
+                  productDispatch({
+                    type: "CATEGORY",
+                    payload: { value: e.target.value },
+                  })
+                }
+              />{" "}
+              Men & Women
+            </label>
+          </div>
+        </div>
+
+        <div className="filter-wrapper">
+          <div className="filter-title">Rating</div>
+          <div className="filter-value filter-rating">
+            <label>
+              <input
+                type="radio"
+                name="rating"
+                checked={productState.rating === 4}
+                onChange={() =>
+                  productDispatch({ type: "RATING", payload: { rating: 4 } })
+                }
+              />{" "}
+              4 <i className="fa-solid fa-star"></i> & above
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="rating"
+                checked={productState.rating === 3}
+                onChange={() =>
+                  productDispatch({ type: "RATING", payload: { rating: 3 } })
+                }
+              />{" "}
+              3 <i className="fa-solid fa-star"></i> & above
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="rating"
+                checked={productState.rating === 2}
+                onChange={() =>
+                  productDispatch({ type: "RATING", payload: { rating: 2 } })
+                }
+              />{" "}
+              2 <i className="fa-solid fa-star"></i> & above
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="rating"
+                checked={productState.rating === 1}
+                onChange={() =>
+                  productDispatch({ type: "RATING", payload: { rating: 1 } })
+                }
+              />{" "}
+              1 <i className="fa-solid fa-star"></i> & above
+            </label>
+          </div>
+        </div>
+
+        <div className="filter-wrapper">
+          <div className="filter-title">Sort By</div>
+          <div className="filter-value filter-sort">
+            <label>
+              <input
+                type="radio"
+                name="sort"
+                checked={productState.sortBy === "HIGH_TO_LOW"}
+                onChange={() =>
+                  productDispatch({
+                    type: "SORT_BY",
+                    payload: { value: "HIGH_TO_LOW" },
+                  })
+                }
+              />{" "}
+              Price - High to Low
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="sort"
+                checked={productState.sortBy === "LOW_TO_HIGH"}
```

Comment:
You can use a constants file to store all action types

Reply:
Yes, I'll do that. Thanks!